main.go

```diff
@@ -182,6 +215,21 @@ func normalize(this js.Value, args []js.Value) interface{} {
 	return Promise.New(handler)
 }
 
+func getEnclosedName(name string) (string, EnclosingType, bool) {
+	var _name string
+	var _enclosure EnclosingType
+	for _, encloseChar := range getEnclosures() {
+		if strings.HasPrefix(name, encloseChar.open) && strings.HasSuffix(name, encloseChar.close) {
+			_, _name, _ = strings.Cut(name, encloseChar.open)
+			_name, _, _ = strings.Cut(_name, encloseChar.close)
+			_enclosure = encloseChar
+			return _name, _enclosure, true
+		}
+	}
+
+	return name, EnclosingType{open: "", close: ""}, false
```

Comment:
This function takes a name node string and if it is enclosed returns the actual name, the enclosure characters, and a bool indicator.  If the name is not enclosed it returns the original name.